### DISCUSSION TITLE
FAR manager plugin landing page in English: How to write FAR manager plugin

### DISCUSSION THREAD

**Discussion by pmisik** [FarGroup/FarManager#682] [General]
Hi

I have an idea to extend FAR manager by plugin.
Is there any landing page how to write plugin for FAR manager in English?
I have found in Far Manager repository link to https://api.farmanager.com/
English and Russian version differs significantly.
It seems Russian version has better description.

Is there any documentation plugins related in English?
If there is no English documentation related to FAR manager plugin architecture basics, can you point to Russian version if there is any (I can use some translator)?

Thank You.

pm-

**Comment by shmuz** [comment]
> Is there any documentation plugins related in English?

There isn't for Far3. The link you've found is the only official documentation.
Use some online translator and ask questions on forum in case something is unclear.

There are some docs in English related to Lua plugins and macros (see %FARHOME%\Encyclopedia).

**Reply by d4rkmen** [reply]
@shmuz english version is obsolette (like v2 API)

**Reply by shmuz** [reply]
> english version is obsolette (like v2 API)

English version of what? Please give a link or describe.

**Reply by d4rkmen** [reply]
This repo `enc\enc_eng`

**Reply by shmuz** [reply]
Yes this is known.
You can follow my initial advice:
1. Use Google translator
2. Ask questions on Far forum when needed.
